runtime-light/stdlib/misc.h

```diff
@@ -18,5 +18,8 @@ void f$die(const mixed &v = 0);
 
 void reset();
 
-task_t<void> parse_input_query(QueryType query_type);
+task_t<void> accept_incoming_query();
+task_t<void> parse_incoming_query();
```

Comment:
I thought about it, but I haven't decided. Are we want to save functions like `component_accept_stream` so on? But I agree that now it can be merged in one function